index.js

```diff
@@ -55,7 +55,13 @@ function main() {
           // Run goldbach here
           let max_int = Number(processNum.trim())
```

Comment:
This will fix the issue where the last line is 0
```suggestion
         if (processNum == '') { // ignores an empty line
            return;
          }
          let max_int = Number(processNum.trim());
```